ArjunaJTS/integration-jakarta/pom.xml

```diff
@@ -0,0 +1,453 @@
+  <description>Narayana: ArjunaJTS integration (atx) Jakarta</description>
+  <packaging>jar</packaging>
+
+  <properties>
+    <modular.jdk.args>--upgrade-module-path ${project.build.directory}/upgrademodulepath --add-exports java.corba/org.omg.CosTransactions=ALL-UNNAMED --add-exports java.corba/com.sun.corba.se.spi.orb=ALL-UNNAMED --add-exports java.corba/com.sun.corba.se.spi.ior=ALL-UNNAMED --add-exports java.corba/com.sun.corba.se.impl.ior=ALL-UNNAMED</modular.jdk.args>
+    <version.jakarta.transaction.jakarta.transaction-api>2.0.0</version.jakarta.transaction.jakarta.transaction-api>
+    <version.jakarta.resource.jakarta.resource-api>2.0.0</version.jakarta.resource.jakarta.resource-api>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <inherited>false</inherited>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Specification-Title>JTA/JTS</Specification-Title>
+              <Specification-Version>1.2/1.0</Specification-Version>
+              <Specification-Vendor>Oracle</Specification-Vendor>
+              <Implementation-Version>${project.version} (revision: ${buildNumber})</Implementation-Version>
+              <arjuna-scm-revision>${buildNumber}</arjuna-scm-revision>
+              <arjuna-properties-file>jbossts-properties.xml</arjuna-properties-file>
+              <arjuna-builder>JBoss Inc. [${user.name}] ${os.name} ${os.version} ${buildproperty.date}</arjuna-builder>
+            </manifestEntries>
+          </archive>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-test</id>
+            <phase>test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals> 
+            <configuration>
+              <useManifestOnlyJar>false</useManifestOnlyJar>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>buildnumber-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>validate</phase>
+            <goals>
+              <goal>create</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <doCheck>false</doCheck>
+          <doUpdate>false</doUpdate>
+          <shortRevisionLength>5</shortRevisionLength>
+          <revisionOnScmFailure>${project.version}-noscm</revisionOnScmFailure>
+        </configuration>
+      </plugin>
```

Comment:
I kept the maven-jar-plugin and buildnumber-maven-plugin because the build number goes into the jar manifest

Reply:
The two `MANIFEST.MF` look nearly the same. Only difference is the tag `Created-By`. If the `maven-jar-plugin` is removed, the Created-By` tag becomes `Apache Maven x.x.x` otherwise it is `Maven Jar Plugin x.x.x`.